### PR TITLE
[intl-tel-input] Add customContainer in Options

### PR DESCRIPTION
**Pull request by chambo-e**
Please fill in this template.

- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [X] Test the change in your own code. (Compile and run.)
- [X] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [X] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [X] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [X] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

This add a missing property in Options interface 

https://github.com/jackocnr/intl-tel-input/blob/ca94755d5899417c2dc674775ee357b32c2c2fa0/src/js/intlTelInput.js#L23

### CODE REVIEW

File: types/intl-tel-input/index.d.ts [hunk 1]
```diff
@@ -156,6 +156,12 @@ declare namespace intlTelInput {
          */
         customPlaceholder?: ((selectedCountryPlaceholder: string, selectedCountryData: CountryData) => string) | undefined;
 
+        /**
+         * Additional classes to add to the parent div..
+         * Default = ''
```
Comment (peterblazejewicz) [inline types/intl-tel-input/index.d.ts:161]:
```suggestion
         * @default ''
```

Comment (chambo-e) [reply types/intl-tel-input/index.d.ts:161]: @peterblazejewicz I matched the format from the others descriptions, would you like me to also update them or just this one ?

Comment (peterblazejewicz) [reply types/intl-tel-input/index.d.ts:161]: we adding JSDoc for users, not to match readme. There is no source code JSDoc, so, your call, sir.

**Review by peterblazejewicz — APPROVED** [review]
LGTM!

File: types/intl-tel-input/index.d.ts (hunk 1 above)
Comment (chambo-e) [reply types/intl-tel-input/index.d.ts:161]: I meant in this file, every other variables uses the `Default = value``` format :)

**Review by peterblazejewicz — APPROVED** [review]
LGTM!

**Comment by chambo-e** [comment]
Ready to merge

**Pull request merged**